graphql-dgs-extended-validation/src/test/kotlin/com/netflix/graphql/dgs/autoconfig/BeanValidationSizeSmokeTest.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.autoconfig
+
+import com.netflix.graphql.dgs.*
+import graphql.schema.idl.SchemaParser
+import graphql.schema.idl.TypeDefinitionRegistry
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.SpringBootConfiguration
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.test.context.SpringBootTest
+import java.util.*
+
+@SpringBootTest(classes = [BeanValidationSizeSmokeTest.LocalApp::class])
+@EnableAutoConfiguration
+internal class BeanValidationSizeSmokeTest {
+
+    @Autowired
+    lateinit var queryExecutor: DgsQueryExecutor
+
+    @Test
+    fun createPostWithInvalidInput() {
+        val query = "mutation newPost(\$input: CreatePostInput!){ createPost(createPostInput: \$input) }"
+        var variables = mapOf(
+            "input" to mapOf(
+                "title" to "test",
+                "content" to "test content"
+            )
+        )
+        val executionResult = queryExecutor.execute(query, variables)
```

Comment:
Is there a specific error that we can assert and check for?

Reply:
Added detailed error message assert.